src/services/notifications.service.js

```diff
@@ -176,50 +156,69 @@ const sendEmailOfType = async (
     );
     throw err;
   }
-  return emailSent;
+  statusEmitter.emit("incrementCount", "emailNotificationsSucceeded");
+  statusEmitter.emit("setStatus", "mostRecentEmailNotificationSucceeded", true);
+  logEmitter.emit("functionSuccess", "registration.service", "sendEmailOfType");
```

Comment:
This function is not called "sendEmailOfType" anymore, so log calls need to be updated.